lib/awful/widget/watch.lua

```diff
@@ -0,0 +1,84 @@
+---------------------------------------------------------------------------
+--- Watch widget.
```

Comment:
:D - do you have a link?

How should screenshots be linked / provided in the doc?